src/components/__tests__/TestUtils.ts

```diff
@@ -86,6 +86,16 @@ export const modifyLibrary = (baseLibrary: LibraryData, newData: {[key: string]:
   return updatedLibrary;
 };
 
+export const validate = (baseLibrary: LibraryData, validatedAt?: string): LibraryData => {
```

Comment:
A `/**` style comment would be good to know what this is supposed to validate and how.